languages/python/exercises/concept/strings/.docs/instructions.md

```diff
@@ -0,0 +1,45 @@
+
+In this exercise you'll be processing log-lines.
+
+Each log line is a string formatted as follows: `"[<LEVEL>]: <MESSAGE>"`.
+
+There are three different log levels:
+
+- `INFO`
+- `WARNING`
+- `ERROR`
+
+You have three tasks, each of which will take a log line and ask you to do something with it.
+
+### 1. Extract a message from a log line
+
+Implement a function to return a log line's message:
+
+```python
+extract_message("[ERROR]: Invalid operation")
```

Comment:
Sorry, hadn't thought about these until I went back to look at the markdown rendering again, but the code examples really should follow Python style, as if the example we being typed into a Python interactive interpreter.

So:

```python
extract_message("[ERROR]: Invalid operation")
// Returns: "Invalid operation"
```

Should really read:

```python
>>> extract_message("[ERROR]: Invalid operation")
'Invalid operation'
```

Or we're just going to confuse people.

Reply:
Agreed!